bugsnag-android-core/src/main/java/com/bugsnag/android/DeviceData.java

```diff
@@ -115,7 +114,7 @@
         return map;
     }
 
-    Map<String, Object> getDeviceMetaData() {
+    Map<String, Object> getDeviceMetadata() {
```

Comment:
Could the Build.BRAND be extracted in buildInfo as well?

Reply:
Good catch - I've updated `DeviceBuildInfo` to contain this field as well.